### PR TITLE
NF-423 - CONTENT/DESIGN FIX: CYA

### PR DESCRIPTION
**Pull request by SteveSugden**

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -287,7 +287,7 @@ check_answers.claim.entryDetails.accessible = Change the entry details
 check_answers.claim.entryDetails.epu = EPU: {0}
 check_answers.claim.entryDetails.entryNumber = Entry number: {0}
 check_answers.claim.entryDetails.entryDate = Entry date: {0}
-check_answers.claim.claimType = What is the type of claim you are making?
+check_answers.claim.claimType = What type of claim you are making?
```
Comment (rorticus0502) [inline conf/messages:290]: This matches the ticket, but it does not look right with this wording and a question mark IMO

File: conf/messages [hunk 2]
```diff
@@ -312,9 +312,9 @@ check_answers.importer.hasEori.true = Yes
 check_answers.importer.hasEori.false = No
 check_answers.importer.eori = Importer’s EORI
 check_answers.importer.eori.accessible = Change the importer’s Economic Operators Registration and Identification number
-check_answers.importer.businessName = Importer's Business name
+check_answers.importer.businessName = Importer’s Business name
```
Comment (rorticus0502) [inline conf/messages:315]: nice

**Review by rorticus0502 — APPROVED** [review]

File: conf/messages (hunk 2 above)
Comment (SteveSugden) [reply conf/messages:315]: Thanks.   Just noticed - should be `Importer’s business name`

**Review by rorticus0502 — APPROVED** [review]
LGTM

**Pull request merged**